packages/react-magma-dom/src/components/Dropdown/Dropdown.test.js

```diff
@@ -969,7 +969,7 @@ describe('Dropdown', () => {
                 <DropdownExpandableMenuButton>
                   Pasta
                 </DropdownExpandableMenuButton>
-                <DropdownExpandableMenuPanel testId={expandablePanelId}>
+                <DropdownExpandableMenuPanel testId="pancakes">
```

Comment:
Do we want to keep this or was this for testing purposes?

Reply:
Oops, reverted